src/nbl/video/IGPUDescriptorSet.cpp

```diff
@@ -201,18 +223,24 @@ void IGPUDescriptorSet::processCopy(const IGPUDescriptorSet::SCopyDescriptorSet&
         auto* dstDescriptors = copy.dstSet->getDescriptors(type, copy.dstBinding);
         assert(!(!srcDescriptors != !dstDescriptors));
```

Comment:
btw the reason we have the `t` loop, is because we have to "guess the type"

if would be useful to `continue` (skip rest of loop) if `!srcDescriptors` like
```cpp
if (!srcDescriptors)
{
   assert(!dstDescriptors);
   continue;
}
assert(dstDescriptors);
```